yew-macro/tests/macro/html-tag-fail.stderr

```diff
@@ -265,44 +257,43 @@ error[E0277]: the trait bound `std::borrow::Cow<'static, str>: std::convert::Fro
              <std::borrow::Cow<'a, [T]> as std::convert::From<std::vec::Vec<T>>>
              <std::borrow::Cow<'a, std::ffi::CStr> as std::convert::From<&'a std::ffi::CStr>>
            and 11 others
-   = note: required by `std::convert::From::from`
-   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
+   = note: required because of the requirements on the impl of `std::convert::Into<std::borrow::Cow<'static, str>>` for `{integer}`
+   = note: required by `std::convert::Into::into`
```

Comment:
What happened here? Maybe it's a nitpick on my part, but I feel like this error was a lot less messy when requiring `From` rather than `Into`.

Reply:
This isn't something I touched in this PR, this change was part of #1447 and is already in:
https://github.com/yewstack/yew/blob/master/yew-macro/tests/macro/html-tag-fail.stderr#L236-L237

Gotta love diff quirks like this.

Anyhow, it does make more sense to accept `Into` than `From` because of the additional flexibility. The slightly more verbose error message is an acceptable trade-off in my opinion.

Reply:
Ah yeah, you're right, there can be more `Into` implementations than `From` implementations, since `From` implies `Into`, but not vice versa. Unfortunate about the error message, but a worthwhile trade-off, I agree.